src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -96,6 +98,7 @@ public StandardSlackService(StandardSlackServiceBuilder standardSlackServiceBuil
         }
         this.roomIds = standardSlackServiceBuilder.roomId.split("[,; ]+");
         this.replyBroadcast = standardSlackServiceBuilder.replyBroadcast;
+        this.sendAsText = standardSlackServiceBuilder.sendAsText;
```

Comment:
Seems unused here? Are we missing an if statement around unfurl_links/unfurl_media?

Reply:
If that line is removed, the `sendAsText` attribute is never applied.  An if statement around unfurl_links/unfurl_media would make things more clear, but isn't necessary.  If messages are being sent as attachments, which is the default, those parameters are benign. I can see the case for adding the if around them though.

Reply:
Then what's the sendAsText for?

Reply:
this only seems to be used in slacksendstep? does it need to be here?

Reply:
You are both correct, it's not necessary.